model/daal4py-LR/.coveragerc

```diff
@@ -0,0 +1,13 @@
+[run]
+source =
+    daal4py_LR
```

Comment:
Let's make this `model/daal4py/`, since all models using `daal4py` will be under this directory

Reply:
Ah, I thought we only want to wraps LR since the issue said "figure out how to do LR with `daal4py`"